html/script.js

```diff
@@ -427,6 +427,11 @@ QBClothing.Open = function(data) {
 
             if (menu.label == "Clothing") {
                 $("#faceoption").css("display", "none");
+                $('div[data-type="model"]').css("display", "none");
+                $('div[data-type="face"]').css("display", "none");
+                $('div[data-type="face2"]').css("display", "none");
+                $('div[data-type="facemix"]').css("display", "none");
+                $('div[data-type="model"]').css("display", "none");
```

Comment:
Looks like you've got a duplicate line here.
Could you remove this, or correct the data type if it's meant to be another type?

Reply:
It's removed